contrib/completion/bash/docker

```diff
@@ -3842,7 +3843,7 @@ _docker_service_update_and_create() {
 			__docker_nospace
 			return
 			;;
-		--group|--group-add|--group-rm)
+		--group-add|--group-rm)
```

Comment:
With this change, `docker service create --group <tab>` no longer completes groups.

Reply:
It still does in my case, can you confirm? 

```
source contrib/completion/bash/docker
source ~/.bash_profile

docker service create --g
bash-3.2$ docker service create --g
--generic-resource  --group             
bash-3.2$ docker service create --g
--generic-resource  --group             
bash-3.2$ docker service create --g
```

Reply:
@rchenzheng
> With this change, `docker service create --group <tab>` no longer completes groups.

The issue is about completing the arguments of the `--group` option.

The code above is only called when the preceding argument is either `--group`, `--group-add` or `--group-rm`. In the context of `service create`, `--group` is still valid and needs support for completing its arguments.

Reply:
Got it! Sorry, it's been a busy day